lib/auth/auth.go

```diff
@@ -190,6 +190,18 @@ func NewServer(cfg *InitConfig, opts ...ServerOption) (*Server, error) {
 	if cfg.UserGroups == nil {
 		cfg.UserGroups = local.NewUserGroupService(cfg.Backend)
 	}
+	if cfg.OktaImportRules == nil {
+		cfg.OktaImportRules, err = local.NewOktaService(cfg.Backend)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+	}
+	if cfg.OktaAssignments == nil {
```

Comment:
can't we use the same backend as OktaImportRules and rename it to `Okta`?
It looks like `local.NewOktaService` already creates two different `generic.NewService`, one per type (OktaAssignments and OktaImportRules)